### DISCUSSION TITLE
How to reduce the amount of times the PRs get "up-to-date" with base branch.

### DISCUSSION THREAD

**Discussion by nsalfos** [renovatebot/renovate#26872] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### What is your question?

tldr: **is there any way to reduce how often Renovate will try to push the updates from the main branch?** 
-

After reducing the scope in which we use renovate, we have currently a few PRs opened to update just one helm-chart in a few locations. 
The thing is that every time there is any update to the main branch (base branch) renovate merge main into its branch to "keep up-to-date with base branch" and the repository has several updates per day, which trigger workflows to run on renovate opened PRs dozens/hundreds of times. 
I've tried configuring  

```
  "schedule": [
    "every 14 days on mondays"
  ],
```
but a few minutes after having that in place i received the notification of renovate pushing a new commit to its branch. 

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by RahulGautamSingh** [comment, marked as answer]
https://docs.renovatebot.com/configuration-options/#rebasewhen
You can check this option. Maybe `conflicted` is the best fr you.

**Reply by nsalfos** [reply]
It seems the schedule did work but impact might've been a bit later than expected. 
i still configured `rebaseWhen=conflicted` so i can manually update the branch in case we're mergin them.
ty.
